Lib/fontTools/merge.py

```diff
@@ -424,9 +424,15 @@ def _glyphsAreSame(glyphSet1, glyphSet2, glyph1, glyph2):
 	g2 = glyphSet2[glyph2]
 	g1.draw(pen1)
 	g2.draw(pen2)
-	return (pen1.value == pen2.value and
-		g1.width == g2.width and
-		(not hasattr(g1, 'height') or g1.height == g2.height))
+	if pen1.value != pen2.value:
+		return False
+	tolerance = .05 if pen1.value else .20
```

Comment:
Not finding the tolerence change entirely intuitive, a comment would help. Your IM idea of ignoring things like CR that never render also makes sense.

Also per IM, warning here would be nice

Reply:
Will add comments.

Reply:
could even make that tolerance an option, the numbers as such look quite arbitrary